keps/sig-node/2413-seccomp-by-default/README.md

```diff
@@ -143,14 +143,31 @@ be extended to cover the new behavior if enabled.
 
 #### Alpha
 
-- [ ] Implement the new feature gate and kubelet configuration
-- [ ] Ensure proper tests are in place
-- [ ] Update documentation to make the feature visible
+- [x] Implement the new feature gate and kubelet configuration
+- [x] Ensure proper tests are in place
+- [x] Update documentation to make the feature visible
 
 #### Alpha to Beta Graduation
 
-- [ ] Enable the feature per default
-- [ ] No major bugs reported in the previous cycle
+- [x] Enable the feature per default
+- [x] Making the feature API aware
+- [x] No major bugs reported in the previous cycle
+
+##### API Awareness
+
+We add conversion methods to the API Server with the graduation of the feature
+to beta. This means that the Kubelet feature gate `SeccompDefault` will be
+enabled by default, while users still have to enable the kubelet configuration
+`SeccompDefault` or use the `--seccomp-default` CLI flag.
```

Comment:
@liggitt @dims from a technical perspective, assuming that the conversion of new workloads will be done on creation in the API Server: How could the information flow that the feature is enabled in the kubelet to the API Server?

Reply:
cc @kubernetes/sig-architecture @kubernetes/sig-node-proposals any thoughts on that?

Reply:
I'm dropping the API awareness to keep simplicity.